tests/edit/test_format_string_upper_and_clean.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from chispa import assert_df_equality
+
+from cishouseholds.edit import format_string_upper_and_clean
+
+
+@pytest.fixture
+def input_df(spark_session):
```

Comment:
I was wondering if the dataframes should be defined without being functions for `input_df `and `expected_df `the way its done in all the other `test_ ` functions for consistency.